src/style.css

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 198px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.card__item-image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__product-description {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 16px 18px;
+}
+
+.card__product-text {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  padding-bottom: 5px;
+  color: hsl(234, 80%, 12%);
+}
+
+.card__product-code {
+  font-size: 10px;
+  font-weight: 400;
+  color: rgb(97, 96, 112);
+}
+
+.card__rating {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  color: rgb(6, 11, 53);
+  padding-right: 16px;
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding-bottom: 16px;
+
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: rgb(97, 96, 112);
+  padding-left: 16px;
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: 700;
+  color: rgb(6, 11, 53);
+  padding-right: 16px;
+  line-height: 18px;
+}
+
+.card__buy-buttom_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  color: #fff;
+  background: rgb(0, 172, 220);
+  font-size: 14px;
+  border-radius: 5px;
+  font-weight: 700;
+  padding: 12px 56px;
+  text-transform: uppercase;
+}
+
+.card__buy-buttom_link:hover {
+  background: #fff;
+  color: rgb(0, 172, 220);
+  border: 1px solid;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover.

You could add border in line 99 for example.